internal/manager/manager.go

```diff
@@ -117,12 +108,18 @@ func newMgr(logger logr.Logger, opts *Options, isController, isReconciler bool)
 	}
 
 	if isReconciler {
-		mgrOpts.Cache.ByObject[&apiv1.Composition{}] = cache.ByObject{
-			Namespaces: map[string]cache.Config{
-				opts.CompositionNamespace: {
-					LabelSelector: opts.CompositionSelector,
+		if opts.CompositionNamespace != cache.AllNamespaces {
+			mgrOpts.Cache.ByObject[&apiv1.Composition{}] = cache.ByObject{
+				Namespaces: map[string]cache.Config{
+					opts.CompositionNamespace: {
```

Comment:
For some reason, setting this value to `AllNamespaces` causes "cache not configured for this namespace" errors, if I understand the implementation correctly, it shouldn't. This works for now, but suggestions are welcome, it should only be a problem in testing code anyways...

Reply:
Yeah I'm not sure that's expected behavior from controller-runtime. In any case though this workaround isn't bad

(feel free to defer this and just add a TODO) We should add a test case to make sure the workaround doesn't regress in case the symphony test changes to no longer cover this functionality.

Reply:
Added TODO.